### PR TITLE
MorePage Update

### LINKED ISSUE #203: Update MorePage to Match Figma Design with improved Usability.
**Description**

The current MorePage is filled with text, making it confusing for users. Update the MorePage to align with the Figma design for a more user-friendly experience.

**Current MorePage:**
<div> Screenshot 2024-11-19 at 11 22 53 AM Screenshot 2024-11-19 at 11 23 09 AM </div>

**Expected MorePage:**
<div> Screenshot 2024-11-19 at 11 18 25 AM Screenshot 2024-11-19 at 11 18 35 AM </div>

**Note: No changes are required for the bottomTabBar.**

**Figma URL:**
https://www.figma.com/proto/aT6Bxx1QLkyMqN9EnWUw7G/Untitled?page-id=0%3A1&node-id=70-1600&node-type=frame&viewport=539%2C-6891%2C1.1&t=OjX4IFbhJq6fllHc-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=1%3A3&show-proto-sidebar=1

**Tasks**

- [x] Locate the MorePage and update its code to match the Figma design.
- [x] Write responsive code that adjusts seamlessly across all devices.
- [x] Implement test cases for the updated MorePage.
- [x] Run the app on both emulator/simulator and physical devices to verify functionality.

**Acceptance Criteria**

- [x] The MorePage matches the design shown in the Figma.
- [x] All test cases pass successfully.
- [x] The app runs smoothly on both emulator/simulator and physical devices.

### PR DESCRIPTION
**Pull request by AmarHadzic**
Fixes #203 

**What was changed**
Updated the more page to reflect the Figma design. The more page was given a more modern look and cleaner layout. Now the more page is more structured and looks complete. This improves usability and user experience.

**Why was it changed**
Previously, the more page included too much text and looked plain, leading to a poor user experience. After the updates, the more page is visually appealing, consistent, and easy to navigate, leading to an improved user-experience.

**How was it changed**
The ```MorePage.tsx``` was completely rewritten based on the Figma design. A ```TeamPage.tsx``` file was created to display the team in a nice consistent format. Also, the ```AppNavigation.tsx``` file was edited to support go back and forth between the More page and its sub pages.

### CODE REVIEW

**Review by rcAsironman — CHANGES_REQUESTED** [review]
@AmarHadzic, good work! However, please follow the Figma design as mentioned in the GitHub issue description.

Changes required:

- [ ] Add an automatic carousel.
- [ ] The dark mode and light mode toggle should reflect the design exactly as it appears in Figma.

Expected output:

https://github.com/user-attachments/assets/4f80209f-eb51-4ebf-981b-07942c3070e9

**Review by rcAsironman — CHANGES_REQUESTED** [review]
@AmarHadzicI found a few alignment issues in your work. On Android, the content in the MorePage overflows and occupies the status bar. Also, on iOS, when I scroll through the options, they overlap with the image carousel. I'll attach the screenshot and screen recording below.

Screenshot_20241202-135102

https://github.com/user-attachments/assets/3c4e793e-6990-4bbe-8ca1-03c8433fe74d

**Review by rcAsironman — APPROVED** [review]
lgtm;

File: __tests__/MoreScreen.test.tsx [hunk 1]
```diff
@@ -64,18 +72,17 @@ beforeEach(() => {
 });
 
 describe('MorePage', () => {
-  it('toggles dark mode correctly', () => {
```
Comment (yashb196) [inline __tests__/MoreScreen.test.tsx:67]: You removed the test case completely why ?

**Review by yashb196 — APPROVED** [review]

File: __tests__/MoreScreen.test.tsx (hunk 1 above)
Comment (AmarHadzic) [reply __tests__/MoreScreen.test.tsx:67]: The more page was completely written so old test cases did not work anymore. I also needed to create a whole new test file.

**Pull request merged**